test/runtests.jl

```diff
@@ -216,7 +223,11 @@ for useio in (false, true)
                 @test x == case.x
             end
             @test code == case.code
-            @test tlen == case.tlen
+            if Parsers.quoted(code)
+                @test tlen == ncodeunits(str)
```

Comment:
i should probably add a comment here:
- we don't check `case.tlen` for quoted input, because our test code replaces the quoted chars in the input with (poentially) multiple characters
- but also for this to be the right check, the testcases have to be limited to inputs without characters after a delimiter, which they are right now, but we might add such testcases in the future so we should probably make the assumption explicit